resources/core/charts/console-backend-service/templates/cluster-role.yaml

```diff
@@ -1,52 +1,52 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: {{ template "fullname" . }}
-  labels:
-    app: {{ template "name" . }}
+    name: {{ template "fullname" . }}
+    labels:
+        app: {{ template "name" . }}
 rules:
-  - apiGroups: ["authorization.k8s.io"]
-    resources: ["subjectaccessreviews"]
-    verbs: ["get", "list", "watch", "create", "delete"]
-  - apiGroups: [""]
-    resources: ["*"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: [""]
-    resources: ["pods","services"]
-    verbs: ["get", "list", "watch", "update", "delete"]
-  - apiGroups: ["extensions", "apps"]
-    resources: ["deployments"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["apps"]
-    resources: ["statefulsets"]
-    verbs: ["list", "watch"]
-  - apiGroups: ["apps"]
-    resources: ["replicasets"]
-    verbs: ["list", "watch", "update", "delete"]
-  - apiGroups: ["servicecatalog.k8s.io"]
-    resources: ["*"]
-    verbs: ["get", "list", "watch", "create", "delete"]
-  - apiGroups: ["servicecatalog.kyma-project.io"]
-    resources: ["servicebindingusages"]
-    verbs: ["get", "list", "watch", "create", "delete"]
-  - apiGroups: ["servicecatalog.kyma-project.io"]
-    resources: ["usagekinds"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["applicationconnector.kyma-project.io"]
-    resources: ["applications", "applicationmappings", "eventactivations"]
-    verbs: ["get", "list", "watch", "create", "delete", "update"]
-  - apiGroups: ["kubeless.io"]
-    resources: ["functions"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["gateway.kyma-project.io"]
-    resources: ["apis"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["authentication.kyma-project.io"]
-    resources: ["idppresets"]
-    verbs: ["get", "list", "watch", "create", "delete"]
-  - apiGroups: ["ui.kyma-project.io"]
-    resources: ["backendmodules"]
-    verbs: ["get", "list", "watch"]
-  - apiGroups: ["*"]
-    resources: ["*"]
-    verbs: ["create"]
\ No newline at end of file
+    - apiGroups: ["authorization.k8s.io"]
+      resources: ["subjectaccessreviews"]
+      verbs: ["get", "list", "watch", "create", "delete"]
+    - apiGroups: [""]
+      resources: ["*"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: [""]
+      resources: ["pods","services","secrets"]
+      verbs: ["get", "list", "watch", "update", "delete"]
+    - apiGroups: ["extensions", "apps"]
+      resources: ["deployments"]
+      verbs: ["get", "list", "watch"]
+    - apiGroups: ["apps"]
+      resources: ["statefulsets"]
```

Comment:
Move all changes within the `resources` directory to image bump pull request, to avoid running integration tests twice.